src/dbt/kipptaf/models/performance_management/rpt_appsheet__leadership_development_assignments.sql

```diff
@@ -0,0 +1,57 @@
+with
+    assignment_group as (
+        select
+            employee_number,
+            if(
+                job_title in (
+                    'Assistant School Leader',
+                    'Assistant School Leader, SPED',
+                    'Assistant School Leader, School Culture',
+                    'School Leader',
+                    'School Leader in Residence',
+                    'Head of Schools',
+                    'Head of Schools in Residence',
+                    'Director School Operations',
+                    'Director Campus Operations',
+                    'Managing Director of School Operations'
+                ),
+                job_title,
+                'CMO and Other Leaders'
+            ) as route,
+        from {{ ref("base_people__staff_roster") }}
+        where assignment_status in ('Active', 'Leave')
+    )
+select
+
+    sr.employee_number,
+    sr.google_email,
+
+    sr2.google_email as google_email_manager,
```

Comment:
I think this already exists as its own column on `base_people__staff_roster`

Reply:
is it called something else? 
![image](https://github.com/TEAMSchools/teamster/assets/68877416/1908324b-f2fc-4a6f-b69b-b3bc8171bc71)